src/main/java/org/semux/gui/panel/ReceivePanel.java

```diff
@@ -304,6 +317,19 @@ protected void copyAddress() {
         }
     }
 
+    /**
+     * Processes the COPY_TO_ADDRESS_BOOK event
+     */
+    protected void copyToAddressBook() {
+        WalletAccount acc = getSelectedAccount();
+        String address = Hex.PREF + acc.getKey().toAddressString();
+        AddressBookEntry entry = new AddressBookEntry("", address);
```

Comment:
This needs to check if they've already added an address book entry for this address.

We don't want this to inadvertantly delete a label they have already set?